tests/foreman/cli/test_contenthost.py

```diff
@@ -588,16 +532,16 @@ def test_contenthost_package_upgrade(self):
         @Assert: Package was successfully upgraded
 
         """
-        self.vm.run('yum install -y {}'.format(FAKE_1_CUSTOM_PACKAGE))
+        self.client.run('yum install -y {}'.format(FAKE_1_CUSTOM_PACKAGE))
```

Comment:
I'd vote that we update this now as it won't take more than a couple of minutes to add the index.